Lib/threading.py

```diff
@@ -1200,6 +1200,9 @@ def excepthook(args, /):
         stderr.flush()
 
 
+__excepthook__ = excepthook
```

Comment:
Would you mind to add a short comment to explain the purpose of this copy? Like:

```suggestion
# Original value of threading.excepthook
__excepthook__ = excepthook
```